releasenotes/notes/otlp-use-min-max-histograms-5c01fe14ef69e127.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    OTLP: Use minimum and maximum values from cumulative OTLP Histograms. Values are used only when we can assume they are from the last time window or otherwise to clamp estimates.
```

Comment:
```suggestion
    OTLP: Use minimum and maximum values from cumulative OTLP Histograms. Values are used only when they are from the last time window, or otherwise to clamp estimates.
```